easybuild/easyconfigs/h/HTSlib/HTSlib-1.4.1-foss-2016a.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# 1.4 modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'HTSlib'
+version = '1.4.1'
+
+homepage = "http://www.htslib.org/"
+description = """A C library for reading/writing high-throughput sequencing data.
+ This package includes the utilities bgzip and tabix"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+checksums = ['e549fee04d95c1cf8f46347e9e9bbd93']
```

Comment:
@FokkeDijkstra Do you mind making this a `SHA256` checksum? Recent EasyBuild versions can auto-detect whether the checksum is `MD5` or `SHA256`, cfr. http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#checksums